src/stretch_kinematics_plugin.cpp

```diff
@@ -0,0 +1,313 @@
+#include <stretch_kinematics_plugin/stretch_kinematics_plugin.h>
+
+#include <tf2/transform_datatypes.h>
+#include <tf2_eigen/tf2_eigen.h>
+
+namespace stretch_kinematics_plugin
+{
+static rclcpp::Logger LOGGER = rclcpp::get_logger("stretch_kinematics_plugin");
+
+StretchKinematicsPlugin::StretchKinematicsPlugin() : initialized_(false)
```

Comment:
Is this considered more idiomatic member initialization than using `bool initialized_{false}` in the class declaration? I am just curious what is considered good practice.

Reply:
Yes, that's right